src/editor.rs

```diff
@@ -697,7 +701,7 @@ impl<'a, W: Write> Editor<'a, W> {
                 buf_num_remaining_bytes = 0;
             } else {
                 buf_num_remaining_bytes -= line.len();
-                write!(self.out, "{}", line)?;
+                write!(self.out, "{}", (self.closure)(line))?;
```

Comment:
I think you can invoke it like `self.closure(line)`, but I'm not sure

Reply:
Sadly, no.

Reply:
o ok